src/typeDefs/errors/validatePaginationArgs.ts

```diff
@@ -0,0 +1,36 @@
+import { gql } from "apollo-server-core";
+
+export const paginationErrors = gql`
+  union PaginationError =
+      PaginationArgsError
+    | MissingArguments
+    | IncorrectPairingOfArguments
+    | FetchLimitExceeded
+    | IncorrectCursor
+
+  interface PaginationArgsError {
```

Comment:
I think a much better interface would be this
```
interface Error {
  message: String!
  path: [String!]!
}
```

Reply:
Got it!